src/datasources/account/account.datasource.spec.ts

```diff
@@ -13,19 +13,37 @@ import {
 import { accountBuilder } from '@/domain/account/entities/__tests__/account.builder';
 import { verificationCodeBuilder } from '@/domain/account/entities/__tests__/verification-code.builder';
 import { getAddress } from 'viem';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as process from 'process';
```

Comment:
I think this is not needed, `process` is a global object: https://nodejs.org/docs/latest-v20.x/api/globals.html#process

Reply:
Fixed in [765de50](https://github.com/safe-global/safe-client-gateway/pull/1235/commits/765de5047d9dced2c9b007dc2c015a71ba9a4423)